CONTRIBUTING.md

```diff
@@ -21,6 +21,8 @@ documentation; new examples and miniapps; HPC performance improvements; etc.
 MFEM is distributed under the terms of the BSD-3 license. All new contributions
 must be made under this license.
 
+Note also that MFEM has a [Code of Conduct](CODE_OF_CONDUCT.md). By participating in the MFEM community, you agree to abide by its rules.
```

Comment:
As a very minor comment, this line does not match the linewidth of this and other `.md` files.

Reply:
Fixed :)

Reply:
Great to see you are still around @barker29